rest-service/manager_rest/rest/resources_v3_1/__init__.py

```diff
@@ -93,5 +93,6 @@
 )
 
 from .labels import (                           # NOQA
-    DeploymentsLabels
+    DeploymentsLabels,
+    DeploymentsLabelsKey
```

Comment:
More commas. Yes, I know it's a style choice- but if you'd done it on the previous PR you'd not need to show an extra +1-1 on the git changes on this commit.

Reply:
Right, changed too.